docs/developer/permissions.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Permissions
+sidebar_label: Permissions
+---
+
+## User permissions
+
+Instead of assigning permissions directly to the user, we define them on a by-group basis. Organizing access rights into [Groups](developer/api-reference/objects/group.mdx) helps in determining staff users' roles in the organization.
```

Comment:
```suggestion
Instead of assigning permissions directly to the user, we define them on a group basis. Organizing access rights in [Groups](developer/api-reference/objects/group.mdx) helps in defining the roles of team members.
```